src/types/line.js

```diff
@@ -244,27 +247,17 @@ function drawLabel(ctx, line, chartArea) {
   }
 
   ctx.fillStyle = label.color;
-  if (isArray(label.content)) {
+  if (content instanceof Image) {
+    const x = -(width / 2) + xPadding + borderWidth / 2;
+    const y = -(height / 2) + yPadding + borderWidth / 2;
+    ctx.drawImage(content, x, y, width - (2 * xPadding) - borderWidth, height - (2 * yPadding) - borderWidth);
+  } else {
+    const labels = isArray(content) ? content : [content];
+    ctx.textBaseline = 'top';
     ctx.textAlign = label.textAlign;
     const x = calculateLabelXAlignment(label, width);
-    let textYPosition = -(height / 2) + label.yPadding;
-    for (let i = 0; i < label.content.length; i++) {
-      ctx.textBaseline = 'top';
-      ctx.fillText(
-        label.content[i],
-        x,
-        textYPosition
-      );
-      textYPosition += label.font.size + label.yPadding;
-    }
-  } else if (label.content instanceof Image) {
-    const x = -(width / 2) + label.xPadding;
-    const y = -(height / 2) + label.yPadding;
-    ctx.drawImage(label.content, x, y, width - (2 * label.xPadding), height - (2 * label.yPadding));
-  } else {
-    ctx.textAlign = 'center';
-    ctx.textBaseline = 'middle';
-    ctx.fillText(label.content, 0, 0);
+    // adds 1.5 because the baseline to top, add 3 pixels from the line for normal letters
+    labels.forEach((l, i) => ctx.fillText(l, x, -(height / 2) + yPadding + (borderWidth / 2) + 1.5 + (i * font.lineHeight)));
```

Comment:
> Where did you get the 3px?

Because, having a look to the picture you published here, I see 2/3 px from baseline and the first pixel on top of a char. Anyway I can remove it.

> Would be simple if calculated something like "topY" before and used topY + i * font.lineHeight here.

Yes, of course . I'm gonna do.

> Also why borderWidth/2?

Usually (but please correct me if I'm wrong) the border width (lineWidth), at a specific location, should have half of the pixels on top of the coordinates.

See below, with the following config:

```javascript
label: {
  position: 'start',
  backgroundColor: 'red',
  content: 'This is a long test label',
  enabled: true,
  borderWidth: 4,
  borderDash: [2, 3],
  borderDashOffset: 0,
  borderColor: 'black'
},
```

![borderWidth](https://user-images.githubusercontent.com/11741250/141357741-301c6fba-5d3e-4316-88c8-8c239708b286.png)

Reply:
Oh right, the border is drawn 50% over the rect, thats why.

Reply:
The text looks (to me) to be too high in that image, is that with the +1.5?

Reply:
Y

> 
> 
> The text looks (to me) to be too high in that image, is that with the +1.5?

yes, it is

Reply:
Should I re-apply it?

Reply:
I'll play with this locally some and get back to you.